index.js

```diff
@@ -21,17 +21,15 @@ var sortStrFormat = /^([+-]|)([\w.]+)(-|)$/;
  *   input was invalid.
  */
 function convertOneSortStrFieldToSQL(sortStr) {
-  'use strict';
   var parsed = sortStr.match(sortStrFormat);
   if (parsed === null) {
     return '';
   }
 
-  var field = parsed[2];
-  var order = parsed[1] === '-' ? 'DESC' : 'ASC';
-  var nulls = parsed[3] === '-' ? 'NULLS FIRST' : 'NULLS LAST';
+  var order = parsed[1] === '-' ? ' DESC' : ' ASC';
+  var nulls = parsed[3] === '-' ? ' NULLS FIRST' : ' NULLS LAST';
 
-  return field + ' ' + order + ' ' + nulls;
+  return parsed[2] + order + nulls;
```

Comment:
This really just makes it harder to read (I'm also quite sure it doesn't matter for performance)